### DISCUSSION TITLE
在iPad设备上文件打不开

### DISCUSSION THREAD

**Discussion by zero-vv** [1Panel-dev/1Panel#6462] [Q&A]
### 联系方式

526666505@qq.com

### 1Panel 版本

1.10.15-beta.1

### 问题描述

在 ipad 设备上文件打不开，手机和电脑可以正常访问，手动输入files路径显示404

### 重现步骤

在iPad设备上点击文件

**Comment by see-more** [comment]
文件路径为`/hosts/files`而不是`/files`
![image](https://github.com/user-attachments/assets/3a70590f-c668-49fc-bd1e-5c2efe54cd42)

**Comment by zero-vv** [comment]
> 文件路径为`/hosts/files`而不是`/files`
> ![图像](https://private-user-images.githubusercontent.com/107225647/366382260-3a70590f-c668-49fc-bd1e-5c2efe54cd42.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjYwNDYyNDcsIm5iZiI6MTcyNjA0NTk0NywicGF0aCI6Ii8xMDcyMjU2NDcvMzY2MzgyMjYwLTNhNzA1OTBmLWM2NjgtNDlmYy1iZDFlLTVjMmVmZTU0Y2Q0Mi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTExJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkxMVQwOTEyMjdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00OTU1MWY5MTgxYzJiNTQ1NWUwMTlmYmFkYjQwMmU3MDA3YjM2ZGNiOWYzYjU3MGY4ZDk2ZjFjZWIxNGU5OGNiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.FG-_991Jpq3l9_8Zxm50KRutRO0UwkruuDPXobAqlT8)

输入的就是hosts/files，问题只在iPad设备上，其他设备正常

**Comment by wanghe-fit2cloud** [comment]
刚刚在 iPad 上测试，未能复现上述问题；在 mac 的 Chrome 浏览器中模拟 iPad 也同样未复现问题。

**Reply by zero-vv** [reply]
https://media-gdgz-fy-person.gd5oss.ctyunxs.cn/PERSONCLOUD/3eae3d7f-746d-4948-a27b-024cb15d67b9.MOV?x-amz-CLIENTTYPEIN=PC&AWSAccessKeyId=g6jU1T3TkAbPKf5ouH5d&x-amz-userLevel=100&x-amz-limitrate=51200&response-content-type=video/quicktime&x-amz-UID=370135704&response-content-disposition=attachment%3Bfilename%3D%22IMG_2324.MOV%22%3Bfilename*%3DUTF-8%27%27IMG_2324.MOV&x-amz-CLIENTNETWORK=UNKNOWN&x-amz-CLOUDTYPEIN=PERSON&Signature=bOEx4VCF5p%2BPWPED3VzBgb%2B85UI%3D&Expires=1726063785&x-amz-FSIZE=47697752&x-amz-UFID=523171155407133256            这个是从稳定版更新至最新beta版bug复现视频链接

**Reply by wanghe-fit2cloud** [reply]
> 在 ipad 设备上文件打不开，手机和电脑可以正常访问

可以尝试更换 iPad 上的浏览器再测试一下。

**Reply by zero-vv** [reply]
使用edge和夸克浏览器出现同样的问题

**Reply by zero-vv** [reply]
[https://pan.vvvxi.com/d/%E5%9B%BE%E7%89%87/IMG_2324.MOV](url)我看那链接打不开了重新发一个

**Reply by wanghe-fit2cloud** [reply]
检查一下浏览器控制台有没有报错、网络请求有没有出现 Error 等。

**Reply by zero-vv** [reply]
在ipad上的浏览器应该是没有这类功能的吧，我用别的浏览器使用代理访问清除浏览器缓存都不行，设备是ipadpro2022 16.1.1系统，服务器是debian12，在两台服务器中都出现了同样的问题，在稳定版就不会，

**Reply by zero-vv** [reply]
回退稳定版就正常了

**Reply by wanghe-fit2cloud** [reply]
1.10.15 版本又发布了两个 beta 版本，可以升级到最新版本再试一下。
> 我在多台设备上都未能复现这个问题，目前还无法准确定位问题原因。

**Reply by zero-vv** [reply]
beta1-3都试了不行，刚新开了台机子装的centos也是同样的问题

**Comment by ssongliu** [comment]
控制台有报错吗？可以加我远程看看这个问题
![image](https://github.com/user-attachments/assets/e0370b44-0434-4d77-b06d-919792eb6241)

**Comment by wanghe-fit2cloud** [comment]
v1.10.16 版本已发布，修复了上述问题。
